app/models/miq_websocket_worker/runner.rb

```diff
@@ -1,3 +1,10 @@
 class MiqWebsocketWorker::Runner < MiqWorker::Runner
   include MiqWebServerRunnerMixin
+
+  def heartbeat(*args)
+    super
```

Comment:
I don't remember if we have to return the result of super here.  I don't think anything checks the return from heartbeat.